finetuner/tailor/keras/projection_head.py

```diff
@@ -0,0 +1,41 @@
+import tensorflow as tf
+
+
+class ProjectionHead(tf.keras.layers.Layer):
+    """Projection head used internally for self-supervised training.
+    It is (by default) a simple 3-layer MLP to be attached on top of embedding model only for training purpose.
+    After training, it should be cut-out from the embedding model.
+    """
+
+    EPSILON = 1e-5
+
+    def __init__(self, in_features: int, output_dim: int = 128, num_layers: int = 3):
```

Comment:
This looks totally correct. 
I think this could be made simpler w/o any branching or the flag variable `is_last_layer`. 
You could simply do: 
```
# For each but the last layer 
for idx in range(num_layers - 1): 
    self.head_layers.append(...) 
    self.head_layers.append(nn.Relu())
# Then append last layer
self.head_layers.append(...)
```
I think this makes it easier to read and reason about, but It is just a style thing, so feel free to reject it.

Reply:
yes! this is something I'm updating now, thanks!